### PR TITLE
Humble edits to "Quick Start" and "Controller Introduction" sections of the english drogon docs.

### PR DESCRIPTION
**Pull request by angstyloop**
Please review my humble edits to the excellent English documentation
as I work my way through the examples. Dracarys!

### CODE REVIEW

File: ENG-03-Quick-Start.md [hunk 1]
```diff
@@ -46,27 +46,49 @@ make
...
 
-Now, we add one simply static file index.html to the Http root path:
+Now, we simply add one static file index.html to the Http root path:
 
 ```shell
 echo '<h1>Hello Drogon!</h1>' >>index.html
 ```
 
-The default root path is `"./"`, this could be modified by config.json also, See [Configuration File](ENG-10-Configuration-File) for more details. Then you can visit this page by URL `"http://localhost"` or`"http://localhost/index.html"` (or the IP of the server where your wepapp running).
+The default root path is `"./"`, the project root (not the build directory). This could also be modified by config.json. See [Configuration File](ENG-10-Configuration-File) for more details. Then you can visit this page by URL `"http://localhost"` or`"http://localhost/index.html"` (or the IP of the server where your wepapp is running).
```
Comment (an-tao) [inline ENG-03-Quick-Start.md]: where `./` represents the working directory of the application, if you run your application in the build directory, './' is the build directory.

**Review by an-tao — CHANGES_REQUESTED** [review]

**Comment by an-tao** [comment]
@angstyloop Thank you very much and sorry for the delay in replying. It looks fine to me except for the description of the `./` directory.

File: ENG-03-Quick-Start.md (hunk 1 above)
Comment (angstyloop) [reply ENG-03-Quick-Start.md]: That makes sense :3 I'll go fix that.

Thanks An!

Comment (angstyloop) [reply ENG-03-Quick-Start.md]: I added a few new commits, and updated the pull request description. The most recent commit fixes my error you described, and provides a little explanation in the commit message.

Besides the fix, the two commits prior to that are just more tiny grammatical fixes and things, that help to polish the already very shiny documentation.

Please let me know if I made any other errors, and thank you for letting me contribute as I play around with Drogon ^.^

**Pull request merged**